components/registration/registration-form.tsx

```diff
@@ -0,0 +1,178 @@
+import React from "react";
+import { InformationCircleIcon, EyeIcon } from "@heroicons/react/outline";
+
+import { InputSelect } from "~/components/ui/forms/input-select";
+import { InputText } from "~/components/ui/forms/input-text";
+import { PrimaryButton as Button } from "~/components/ui/button/primary-button";
+import { FormGroup } from "~/components/ui/forms/form-group";
+import { InputCheckboxCustom } from "~/components/ui/forms/input-checkbox-custom";
+
+export function RegistrationForm() {
+  return (
+    <div>
+      <div className="mb-6">
+        <div className="mb-3">
+          <span>Nama Lengkap</span>
+        </div>
+        <InputText />
+      </div>
+      <div className="mb-6">
+        <div className="mb-3">
+          <span>Nomor Whatsapp yang aktif</span>
+        </div>
+        <FormGroup>
+          <span className="bg-gray-200 px-4 flex items-center rounded-l-md border-2 border-gray-00">
+            +62
+          </span>
+          <InputText
+            className="rounded-l-none block w-full sm:text-sm rounded-md"
+            placeholder="838xxxxxxx"
+            type="text"
+          />
```

Comment:
```suggestion
          <InputText
            className="block w-full sm:text-sm rounded-md"
            isGroupItem
            placeholder="838xxxxxxx"
            type="text"
          />
```

Please use `isGroupItem` for `InputText` inside `FormGroup`.

Reply:
Noted @resir014

need I remove the `rounded-l-none` also ?

because when I remove the class, the input will have rounded on the left side

<img width="1023" alt="Screen Shot 2021-08-11 at 18 37 49" src="https://user-images.githubusercontent.com/22621375/129022558-e8c62a3e-c3e7-4357-abaa-dbf652fe542d.png">

Reply:
If you added `isGroupItem` the rounded corners should automatically adjust. Are you sure you've added it?